docking.js

```diff
@@ -574,6 +574,12 @@ const DockedDash = GObject.registerClass({
             'changed::dance-urgent-applications',
             () => this.dash.resetAppIcons(),
             Utils.SignalsHandlerFlags.CONNECT_AFTER,
+        ], [
+            settings,
+            'changed::hide-tooltip',
+            () => {
+                this.dash.resetAppIcons();
+            },
```

Comment:
I'm not sure this is needed, at the end this should be applied immediately once you hover the icon anyways.

Reply:
You are right, this is not needed. 
Do you plan to accept this PR? Do I need to make a new one without docking.js change?